### PR TITLE
Overall polishment

### PR DESCRIPTION
**Pull request by TomDijkema**
PR solves a lot of minor issues that were occurring within DiSSCover.
Please see the individual commit names for the issue indications.

### CODE REVIEW

File: src/components/search/Search.tsx [hunk 1]
```diff
@@ -59,8 +59,8 @@ const Search = () => {
     const compareMode = useAppSelector(getCompareMode);
     const paginationObject = useAppSelector(getPaginationObject);
     const pageSize = 25;
-    const [pageNumber, setPageNumber] = useState<number>(
-        (paginationObject.page === location.pathname.split('/')[1]) ? paginationObject.pageNumber : 1
+    const [pageNumber, setPageNumber] = useState<number | undefined>(
+        /*(paginationObject.page === location.pathname.split('/')[1]) ? paginationObject.pageNumber :*/ undefined
```
Comment (samleeflang) [inline src/components/search/Search.tsx:63]: Is this comment needed?

File: src/components/specimen/components/contentBlocks/OriginalData.tsx [hunk 2]
```diff
@@ -62,6 +64,46 @@ const OriginalData = () => {
...
+                height: '100%',
+                width: '100%'
+            }
+        },
+        head: {
+            style: {
+                fontSize: '0.875rem !important'
+            }
+        },
+        // rows: {
```
Comment (samleeflang) [inline src/components/specimen/components/contentBlocks/OriginalData.tsx:93]: Can this be removed?

**Review by samleeflang — COMMENTED** [review]
:broccoli:  Couple small comments

File: src/components/general/mediaTypes/ImageViewer.tsx [hunk 3]
```diff
@@ -58,11 +59,8 @@ const ImageViewer = (props: Props) => {
     const [editAnnotation, setEditAnnotation] = useState<ImageAnnotation | null>(null);
 
     const OSDOptions: OpenSeadragon.Options = {
-        prefixUrl: 'https://cdn.jsdelivr.net/npm/openseadragon@3.1/build/openseadragon/images/',
-        tileSources: {
-            type: 'image',
-            url: mediaUrl
-        },
+        prefixUrl: "https://cdn.jsdelivr.net/npm/openseadragon@2.4/build/openseadragon/images/",
```
[the remark this answers is not in the document]
Comment (TomDijkema) [reply src/components/general/mediaTypes/ImageViewer.tsx:62]: Was testing if that would solve the CORS issue, changed it back to the latest one

File: src/components/search/Search.tsx (hunk 1 above)
Comment (TomDijkema) [reply src/components/search/Search.tsx:63]: No, removed

File: src/components/specimen/components/contentBlocks/OriginalData.tsx (hunk 2 above)
Comment (TomDijkema) [reply src/components/specimen/components/contentBlocks/OriginalData.tsx:93]: Yes, removed

**Review by southeo — APPROVED** [review]
:pear: Looks good to me

**Pull request merged**